Code/Framework/AtomCore/AtomCore/Instance/InstanceId.h

```diff
@@ -95,7 +109,12 @@ namespace AZ
         template<class StringType>
         inline void InstanceId::ToString(StringType& result) const
         {
-            result = StringType::format("%s:%x:%p", m_guid.ToString<StringType>().c_str(), m_subId, m_versionId);
+            StringType combinedStr = m_guid.ToString<StringType>();
+            for (const auto subId : m_subIds)
+            {
+                combinedStr += StringType::format(":%x", subId);
+            }
```

Comment:
Suggestion: I believe the AP prints out subIds as decimal numbers, so it might be easier to match up the two if you use %u here.